.drone.yml

```diff
@@ -0,0 +1,51 @@
+kind: pipeline
+name: default
+
+platform:
+    os: linux
+    arch: amd64
+
+steps:
+- name: edelweiss_swc
+  image: mythx.azurecr.io/tools/edelweiss
+  environment:
+    IGNORE_FALSE_POSITIVES:
+      from_secret: IGNORE_FALSE_POSITIVES
+    IGNORE_REGRESSIONS:
+      from_secret: IGNORE_REGRESSIONS
+    AWS_DEFAULT_REGION:
+        from_secret: AWS_DEFAULT_REGION
```

Comment:
why is the indentation on this line different from line 15

Reply:
indents fixed